pkgs/development/libraries/science/math/petsc/default.nix

```diff
@@ -30,23 +53,35 @@ stdenv.mkDerivation rec {
     patchShebangs .
     configureFlagsArray=(
       $configureFlagsArray
-      "--CC=$CC"
-      "--with-cxx=$CXX"
-      "--with-fc=$FC"
-      "--with-mpi=0"
+      ${if !mpiSupport then ''
+        "--CC=$CC"
+        "--with-cxx=$CXX"
+        "--with-fc=$FC"
```

Comment:
Should there be a non-Fortran build? (I don't know nixpkgs priorities; many users don't need Fortran.)